lib/data/2022_hud_lists.json

```diff
@@ -1191,7 +1179,7 @@
       },
       {
         "key": 27,
-        "description": "Interim Housing"
+        "description": "Moved from one HOPWA funded project to HOPWA TH"
```

Comment:
This display string was incorrect on the living_situations list. https://hudhdx.info/Resources/Vendors/HMIS_CSV_Specifications_FY2022_v1.3.pdf